.github/workflows/release.yaml

```diff
@@ -0,0 +1,39 @@
+name: Release Android SDK
+
+on: 
+  workflow_dispatch:
+    branches:
+        - master
+        - develop
```

Comment:
do we need develop branch as well ?

Reply:
Yes, indeed. I removed the option to release from the `develop` branch.